src/app/lib/config/toggles/liveConfig.js

```diff
@@ -44,4 +44,7 @@ export default {
   variantCookie: {
     enabled: true,
   },
+  asyncAds: {
```

Comment:
Do we need these values (if toggle does not exist, then it will always be false by default). Also means less clean up required?

Reply:
good point, will remove